src/app/shared/services/affinity-group.service.ts

```diff
@@ -31,7 +31,8 @@ export class AffinityGroupService extends BaseBackendCachedService<AffinityGroup
 
   public create(params: AffinityGroupCreationData): Observable<AffinityGroup> {
     return super.create(params)
-      .switchMap(job => this.asyncJob.queryJob(job.jobid, this.entity, this.entityModel));
+      .switchMap(job => this.asyncJob.queryJob(job.jobid, this.entity, this.entityModel))
+      .map((job: AsyncJob<any>) => job.jobresult.affinitygroup);
```

Comment:
is it possible to get rid of **any**

Reply:
Due to the fact that we began to use interfaces when switching to ngrx, the services began to work a little differently. In this case, AsyncJob generic class will not work correctly.